### ISSUE TITLE
Can't launch

### ISSUE THREAD

**Issue by philberndt** [Ulauncher/Ulauncher#200]
**Ulauncher version**: v4.0.3.r0
**Linux distro**: Arch Linux x86_64 4.14.13-1-ARCH
**Desktop environment**: Mate

I get the following error when trying to launch

```
/usr/lib/python2.7/site-packages/ulauncher/util/desktop/notification.py:1: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Notify
/usr/lib/python2.7/site-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py:6: PyGIWarning: WebKit2 was imported without specifying a version first. Use gi.require_version('WebKit2', '4.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gio, Gtk, WebKit2, GLib
Traceback (most recent call last):
  File "/usr/bin/ulauncher", line 25, in <module>
    main()
  File "/usr/lib/python2.7/site-packages/ulauncher/main.py", line 82, in main
    if is_wayland() and gdk_backend() != 'x11':
  File "/usr/lib/python2.7/site-packages/ulauncher/config.py", line 63, in is_wayland
    return os.environ['XDG_SESSION_TYPE'].lower() == 'wayland'
  File "/usr/lib/python2.7/UserDict.py", line 40, in __getitem__
    raise KeyError(key)
KeyError: 'XDG_SESSION_TYPE'

```

**Comment by gornostal** [comment]
duplicate of #197 
Fixed in dev. I'll make a stable release now

**Comment by philberndt** [comment]
Yes, dev is working just fine :1st_place_medal:

**Comment by gornostal** [comment]
Fixed in stable release 4.0.4.r1
